LDAR-Sim_inputs.md

```diff
@@ -711,17 +712,6 @@ If you are developing for LDAR-Sim, please adhere to the following rules:
 
 __*Temporary__: Obsolete for V2 methods.
 
-## timesteps
-
-**Data type:** Integer
-
-**Default input:** N/A
-
-**Description:** The number of daily timesteps to use for each simulation run of the program.
-
-**Notes on acquisition:** No data acquisition required.
-
-**Notes of caution:** We recommend running the simulation for several years due to the stochastic nature of LDAR systems and the periods of time over which leaks arise and are repaired. Using 1000 timesteps is a good place to start. Keep in mind that if a spin\_up is used, the number of timesteps must be greater. Also note that the simulation will run until start\_date + timesteps, so keep in mind that the end date cannot extend beyond the available ERA5 weather data, if used.
```

Comment:
Can we transfer this over to the new notes of caution for end_date? This is important. Also, can leave out comment on spin_up as this will be depreciated very soon.